File: example-spark-learn/src/main/java/com/undergrowth/bean/Person.java
package com.undergrowth.bean;

import java.io.Serializable;

/**
 * @author zhangwu
 * @version 1.0.0
 * @description Person
 * @date 2017-11-01-23:41
 */
public class Person implements Serializable {

  private String name;
  private int age;

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public int getAge() {
    return age;
  }

  public void setAge(int age) {
    this.age = age;
  }
}

File: example-scala-learn/src/basic/Person.scala
package basic

/**
  * @description
  * @author zhangwu
  * @date 2017-11-03-23:42
  * @version 1.0.0
  */
class Person {
  var name = ""
  override def toString = getClass.getName + "[name=" + name + "]"
}
